### ISSUE TITLE
Most recent release only works with Python 3.6+

### ISSUE THREAD

**Issue by bricksdont** [mjpost/sacreBLEU#33]
I installed the lastest PyPi release of sacrebleu today, and got the following error:

```python
$ sacrebleu
File "/home/user/mmueller/mtrain-toy-models/venvs/mtrain3/bin/sacrebleu", line 6, in <module>
from sacrebleu import main
File "/home/user/mmueller/mtrain-toy-models/venvs/mtrain3/lib/python3.5/site-packages/sacrebleu.py", line 1144
precisions = "/".join(f"{p:.1f}" for p in self.precisions)
```

After researching for a while, it turns out an expression like this only works with Python 3.6 or higher:

```bash
$ pyenv shell 3.5.0
$ python -c 'precisions = "/".join(f"{p:.1f}" for p in [3, 2, 1])'
  File "<string>", line 1
    precisions = "/".join(f"{p:.1f}" for p in [3, 2, 1])
                                   ^
SyntaxError: invalid syntax
$ pyenv shell 3.6.0
$ python -c 'precisions = "/".join(f"{p:.1f}" for p in [3, 2, 1])'
$ pyenv shell 3.7.0
$ python -c 'precisions = "/".join(f"{p:.1f}" for p in [3, 2, 1])'
```

@Matt Can you reproduce this?

I think as many versions of Python 3 as possible should be supported.

**Comment by bricksdont** [comment]
Just realized this was already reported:

https://github.com/mjpost/sacreBLEU/pull/32

**Comment by mjpost** [comment]
This is added in the repo. I'll include it in the next release. (Do you work from releases or the code?)

**Comment by mjpost** [comment]
Just released v1.3.5 which should restore 3.5 compat.
